codespell_lib/data/dictionary.txt

```diff
@@ -32001,6 +32109,7 @@ lsip->lisp
 lsit->list, slit, sit,
 lsits->lists, slits, sits,
 luckly->luckily
+luetooth->bluetooth
```

Comment:
Perhaps **B**luetooth, as it is a brand name?

Reply:
There were other entries with `bluetooth`, so I capitalized them too.